src/api/users/index.ts

```diff
@@ -1,10 +1,22 @@
 import { defaultFetchConfig, expectResponseCode, getApiUrl } from '../utils'
-import { UserResponse } from './types'
+import { UserResponse, UserResponseCacheEntry } from './types'
+
+const CACHE_TIME_SECONDS = 600
```

Comment:
This could be moved into a cache-time.json file. Especially if we have more caches in the future and they may have different times?

Reply:
But every cache will be bound to an API request method, doesn't it?
In that case, I would argue that having the cache time definition in the same place as the caching logic is more intuitive.

Reply:
I like the idea of changing the cache times in a single file, but I guess it doesn't happen to often.

Reply:
That's the point. And even if you change the times, that will be opening maybe 3-4 files.
The single static links collection is another topic because they're used over the whole project.